src/components/GoalHeader.tsx

```diff
@@ -0,0 +1,99 @@
+import { nullable } from '@taskany/bricks';
+import { ComponentProps, FC, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { GoalByIdReturnType } from '../../trpc/inferredTypes';
+
+import StateSwitch from './StateSwitch';
+import { State } from './State';
+import { IssueStats } from './IssueStats/IssueStats';
+import { IssueTitle } from './IssueTitle';
+
+const IssueHeader = styled.div`
```

Comment:
Must be `StyledGoalHeader`

https://github.com/taskany-inc/issues/blob/main/docs/patterns.md